### PR TITLE
fix es

### PR DESCRIPTION
**Pull request by aliotta**
## 📋 Checklist

- [ ] The PR title is informative to the user experience

### CODE REVIEW

File: charts/astronomer/templates/houston/houston-configmap.yaml [hunk 1]
```diff
@@ -275,21 +275,21 @@ data:
...
+    # skip the configuration for elasticsearch. Houston will fallback
+    # to its default modes, which are false/disabled.
+    # XXX: We may want to decouple these configurations from the
+    # singleNamespace config. At the moment, this is a convenient
+    # helper to require a minimal config change to run the platform in
+    # single namespace mode. Fluentd requires a ClusterRole to
+    # enrich logs with kubernetes metadata. In a single namespace environment,
+    # we're likely not going to get a ClusterRole from admins.
+    {{- if not .Values.global.singleNamespace }}
+      elasticsearch:
```
Comment (jedcunningham) [inline charts/astronomer/templates/houston/houston-configmap.yaml:288]: I'm having trouble working out the change here. Is this still under `airflow.elasticsearch`?

Comment (aliotta) [reply charts/astronomer/templates/houston/houston-configmap.yaml:288]: so currently its not under airflow.elasticsearch but deployments.helm. this change slides it back to deployments

Comment (aliotta) [reply charts/astronomer/templates/houston/houston-configmap.yaml:288]: the default.yaml file in houston is expecting it at the deployments level. And thus the default of host: localhost is not being over written

File: charts/astronomer/templates/commander/commander-role.yaml [hunk 2]
```diff
@@ -43,7 +43,7 @@ rules:
   verbs: ["*"]
 - apiGroups: [""]
   resources: ["persistentvolumeclaims"]
-  verbs: ["create", "delete", "deletecollection", "get", "list", "update", "watch"]
+  verbs: ["create", "delete", "deletecollection", "get", "list", "update", "watch","patch"]
```
Comment (danielhoherd) [inline charts/astronomer/templates/commander/commander-role.yaml:46]:
```suggestion
  verbs: ["create", "delete", "deletecollection", "get", "list", "update", "watch", "patch"]
```

File: charts/astronomer/templates/commander/commander-role.yaml [hunk 3]
```diff
@@ -61,7 +61,7 @@ rules:
   verbs: ["get"]
 - apiGroups: [""]
   resources: ["persistentvolumes"]
-  verbs: ["create", "delete", "get", "list", "watch"]
+  verbs: ["create", "delete", "get", "list", "watch","patch"]
```
Comment (danielhoherd) [inline charts/astronomer/templates/commander/commander-role.yaml:64]:
```suggestion
  verbs: ["create", "delete", "get", "list", "watch", "patch"]
```

**Review by danielhoherd — CHANGES_REQUESTED** [review]
Suggested consistent spacing.

File: charts/astronomer/templates/houston/houston-configmap.yaml [hunk 4]
```diff
@@ -275,21 +275,21 @@ data:
...
+        # If we're in single namespace mode, we will also
+        # skip the configuration for elasticsearch. Houston will fallback
+        # to its default modes, which are false/disabled.
+        # XXX: We may want to decouple these configurations from the
+        # singleNamespace config. At the moment, this is a convenient
+        # helper to require a minimal config change to run the platform in
+        # single namespace mode. Fluentd requires a ClusterRole to
+        # enrich logs with kubernetes metadata. In a single namespace environment,
+        # we're likely not going to get a ClusterRole from admins.
+        {{- if not .Values.global.singleNamespace }}
```
Comment (danielhoherd) [inline charts/astronomer/templates/houston/houston-configmap.yaml:287]: Is this going to work? I think the `{{-` will cause line 288 to be over-indented. We should write tests for this.

Comment (aliotta) [reply charts/astronomer/templates/houston/houston-configmap.yaml:287]: Test added it does work.

**Review by jedcunningham — APPROVED** [review]

**Pull request merged**